cmd/get.go

```diff
@@ -65,6 +67,14 @@ and provides a fast and easy alternative to a package manager.`,
 			dlMode = get.DownloadArkadeDir
 		}
 
+		currentUser, err := user.Current()
```

Comment:
Today I learned about `os/user` 👍 thanks

Perhaps the existing code behind InitUserDir may benefit from using user.Current instead? Are there any drawbacks or reliances on syscalls/C?